src/olympia/addons/models.py

```diff
@@ -2083,6 +2083,11 @@ def transformer(overrides):
     def compat_ranges(self):
         return list(self._compat_ranges.all())
 
+    @property
+    def version_ranges(self):
```

Comment:
Do you need that? Would `source='collapsed_ranges'` have worked in the serializer?

Reply:
oh, it does.  I thought source needed a property for some reason.